### PR TITLE
WP-358

### PR DESCRIPTION
**Pull request by paveldhq**
Refactored Integration tests

### CODE REVIEW

File: tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php [hunk 1]
```diff
@@ -20,50 +20,83 @@
 use Smartling\Submissions\SubmissionManager;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
-abstract class SmartlingUnitTestCaseAbstract extends \WP_UnitTestCase
+
+abstract class SmartlingUnitTestCaseAbstract extends WP_UnitTestCase
 {
-    public function setUp()
+
+    public static function setUpBeforeClass()
```
Comment (PavelLoparev) [inline tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php]: Do we need empty method?

File: tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php [hunk 2]
```diff
@@ -323,5 +375,18 @@ public function getSubmissionById($id)
...
+            'post_status'  => 'publish',
+            'post_type'    => $post_type,
+            'meta_input'   => $meta,
+        ];
+
+        $postId = $this->factory()->post->create_object($template);
 
+        return $postId;
+    }
 }
```
Comment (PavelLoparev) [inline tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php]: Let's enter EOL here and throughout this PR

File: tests/IntegrationTests/tests/LockTranslationTest.php [hunk 3]
```diff
@@ -33,9 +29,9 @@ public function testLockPostFlow()
         $translatedTitle = $this->getContentHelper()->readTargetContent($submission)->getTitle();
 
         $this->editPost([
-            'ID' => $postId,
-            'post_title' => $sourceTitle . ' EDITED.',
-        ]);
+                            'ID'         => $postId,
+                            'post_title' => $sourceTitle . ' EDITED.',
+                        ]);
```
Comment (PavelLoparev) [inline tests/IntegrationTests/tests/LockTranslationTest.php]: Why do we need these indents?

File: tests/IntegrationTests/tests/LockTranslationTest.php [hunk 4]
```diff
@@ -48,9 +44,9 @@ public function testLockPostFlow()
         // and re-upload/re-download it again - previous edited title = new
         // "translated" title.
         $this->editPost([
-            'ID' => $postId,
-            'post_title' => $sourceTitle . ' EDITED ONCE AGAIN.',
-        ]);
+                            'ID'         => $postId,
+                            'post_title' => $sourceTitle . ' EDITED ONCE AGAIN.',
+                        ]);
```
Comment (PavelLoparev) [inline tests/IntegrationTests/tests/LockTranslationTest.php]: Why do we need these indents?

File: tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php (hunk 2 above)
Comment (PavelLoparev) [inline tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php]: Let's enter EOL here and throughout this PR

Comment (PavelLoparev) [inline tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php]: Let's enter EOL here and throughout this PR

File: tests/IntegrationTests/tests/LockTranslationTest.php (hunk 3 above)
Comment (paveldhq) [reply tests/IntegrationTests/tests/LockTranslationTest.php]: just autoformat

File: tests/IntegrationTests/tests/LockTranslationTest.php (hunk 4 above)
Comment (paveldhq) [reply tests/IntegrationTests/tests/LockTranslationTest.php]: just autoformat

File: tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php (hunk 1 above)
Comment (paveldhq) [reply tests/IntegrationTests/SmartlingUnitTestCaseAbstract.php]: removed

**Pull request merged**